evalml/automl/automl_algorithm/automl_algorithm.py

```diff
@@ -52,6 +52,11 @@ def add_result(self, score_to_minimize, pipeline):
             score_to_minimize (float): The score obtained by this pipeline on the primary objective, converted so that lower values indicate better pipelines.
             pipeline (PipelineBase): The trained pipeline object which was used to compute the score.
         """
+        if self.batch_number == 1:
```

Comment:
We use this try/except on batch 1 because this batch contains the pipelines with components initialized on their default args. If this is outside of the search space (hyperparameter ranges) for the components, then we receive a `ValueError`, which we then use to raise a more interpretable `ValueError`.